liquibase-core/src/main/java/liquibase/parser/NamespaceDetails.java

```diff
@@ -5,7 +5,7 @@
 
 public interface NamespaceDetails extends PrioritizedService{
 
-    public static int PRIORITY_EXTENSION = PRIORITY_DATABASE;
+    int PRIORITY_EXTENSION = PRIORITY_DATABASE;
```

Comment:
Is there any functional reason it makes sense to allow multiple copies of `PRIORITY_EXTENSION` in the JVM?

Reply:
no. its just redundant.
https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html

Reply:
Hi, @arturobernalg!  Thanks for doing so much code cleanup for the project - it is awesome! Is this the bit of the documentation that applies? 
```
Every field declaration in the body of an interface is implicitly public, static, and final.
```

I'm not a Java developer so thank you for being kind about answering my questions. :)

Reply:
Hi, exactly. All variables declare in an interface are by default public static final.

Thank you.